### PR TITLE
Update installation page 📖

### PR DESCRIPTION
**Pull request by rav3**
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Code enhancement and update (non-breaking change)
- [ ] Security patch

### CODE REVIEW

File: collections/_get-started/installation.html [hunk 1]
```diff
@@ -1,33 +1,86 @@
...
+</body>
+{% endcapture %}
 
 <div class="page-component">
-    {% include page-component/component-header.html %}
-    
+  {% include page-component/component-header.html %}
+
+  <section name="section-vanillaJS">
+    <h2 class="title fw-bold" id="vanilla-js">For React JS</h2>
```
Comment (haphizi) [inline collections/_get-started/installation.html:40]: change attr name and id to reactJs?

File: _config.yml [hunk 2]
```diff
@@ -15,6 +16,7 @@ search_placeholder: "What are you looking for?"
 react-storybook: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/
```
Comment (clukhei) [inline _config.yml:16]: not sure if this matters now cos later will change again. 

current react-storybook is https://2-0-0-dev.dg2jutx9gylat.amplifyapp.com/?path=/story/install--page

Comment (rav3) [reply _config.yml:16]: lets update this once we confirm the domain names

File: collections/_get-started/installation.html [hunk 3]
```diff
@@ -1,33 +1,86 @@
...
 title: Installation
 layout: layout-sidenav
-description: A dropdown mechanism that allow users to either show or hide related content.
+description: Installation guide that will help you quicky spin up a site using SGDS
 ---
+{% capture reactjs %}
+npm install @govtechsg/sgds-react
+
+//not required if using cdn
+npm install @govtechsg/sgds-govtech bootstrap-icons
```
Comment (clukhei) [inline collections/_get-started/installation.html:10]: npm install @govtechsg/sgds bootstrap-icons 

ok i missed this out on react installation guide too

File: collections/_get-started/installation.html [hunk 4]
```diff
@@ -1,33 +1,86 @@
...
+{% endcapture %}
+{% capture using-node %}
+npm i @govtechsg/sgds bootstrap@5.1.3 bootstrap-icons
+{% endcapture %}
+
+{% capture webpack %}
+import "@govtechsg/sgds/css/sgds.css";
+import 'bootstrap';
+// or get all of the named exports for further usage
```
Comment (clukhei) [inline collections/_get-started/installation.html:19]: Need to import bootstrap? I thought @govtechsg/sgds library already contains bootstrap?

Comment (rav3) [reply collections/_get-started/installation.html:19]: oh yes!! great catch!!

File: collections/_get-started/installation.html (hunk 3 above)
Comment (clukhei) [reply collections/_get-started/installation.html:10]: @rav3 think u missed this out. Other than that I approved

**Review by clukhei — APPROVED** [review]
Left out a change in installation.html , other than that I approve

**Review by haphizi — APPROVED** [review]

**Review by sophiaaachow — APPROVED** [review]

**Pull request merged**